blenderproc/python/types/MeshObjectUtility.py

```diff
@@ -479,14 +479,14 @@ def add_displace_modifier(self, texture: bpy.types.Texture, mid_level: float = 0
                                         'min_vertices_for_subdiv' a Subdivision modifier will be add to the object.
         :param subdiv_level:  Numbers of Subdivisions to perform when rendering. Parameter of Subdivision modifier.
```

Comment:
missing return docu

Reply:
Actually most methods miss this here. Should I go ahead and add them for all?

Reply:
Yes sure, that would be great